### PR TITLE
Rebranding error pages

### PR DESCRIPTION
**Pull request by devramsean0**
## Summary of the problem

This brings the error pages that are served on GitHub pages that are designed for when Heroku is having issues closer to the other error pages. (see #4563)

## Describe your changes

I went about recreating the [500 error page](https://hcb.hackclub.com/500.html) as closely as made sense (notice the lack of the second row in the footer, and the changelog icon among other small things)

image
image

### CODE REVIEW

File: app/controllers/admin_controller.rb [hunk 1]
```diff
@@ -1107,6 +1107,18 @@ def check_deposits
...
+    @invoices = Invoice.where(creator_id: @user.id)
+    @donations = Donation.where(email: @user.email) # Probably not the best way to fetch this but I don't have any other ideas
+    @check_deposits = CheckDeposit.where(created_by_id: @user.id)
+    @increase_checks = IncreaseCheck.where(user_id: @user.id)
+    @lob_checks = Check.where(creator_id: @user.id)
+    @ach_transfers = AchTransfer.where(creator_id: @user.id)
+    @disbursements = Disbursement.where(requested_by_id: @user.id)
+    render layout: "admin"
+  end
+
```
Comment (sampoder) [inline app/controllers/admin_controller.rb]: @devramsean0 looks like a couple of changes form your other PR are in here, cool if they get removed?

**Review by sampoder — APPROVED** [review]
Looks wise, this looks good to me! It might be fun to hide some easter eggs in both of these pages but these do the job perfectly.

**Comment by aramshiva** [comment]
no rickrolls :/ 
~~2/10 stars~~

File: app/controllers/admin_controller.rb (hunk 1 above)
Comment (devramsean0) [reply app/controllers/admin_controller.rb]: Yeah...
Why does this always happen to me :(

**Comment by devramsean0** [comment]
I can't think of any interesting Easter eggs :/
Any ideas? bc that sounds fun

File: docs/application-error.html [hunk 2]
```diff
@@ -1,67 +1,57 @@
...
-</body>
-</html>
+    <head>
+        <title>We're sorry, but something went wrong (500)</title>
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <style>
+            body {
+                text-align: center;
+                font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+  '             Fira Sans', Oxygen, Ubuntu, 'Helvetica Neue', sans-serif;
```
Comment (garyhtou) [inline docs/application-error.html]: Should this `'` be moved so there isn't leading whitespace in the string?

File: docs/application-error.html [hunk 3]
```diff
@@ -1,67 +1,57 @@
...
+        <header>
+            <img class="logo" src="https://hcb.hackclub.com/brand/hcb-icon-icon-original.png" width="40" height="40" />
+            <h1>That didn't work!</h1>
+        </header>
+        <p>We're going to debug what went wrong. To any staff: <em>this is a platform error</em>.</p>
+        <img class="dino" src="https://cloud-4q3ilvs9v-hack-club-bot.vercel.app/0dino_debugging.png" height="400px"/>
+        <br />
+        <span style="color: #8492a6;">Contact the HCB team at <a href="mailto:hcb@hackclub.com">hcb@hackclub.com</a></span>
+    </body>
+</html>
```
Comment (garyhtou) [inline docs/application-error.html]: we always keep a newline at the end of files
```suggestion
</html>

```

File: docs/maintenance-mode.html [hunk 4]
```diff
@@ -1,67 +1,57 @@
...
+        <header>
+            <img class="logo" src="https://hcb.hackclub.com/brand/hcb-icon-icon-original.png" width="40" height="40" />
+            <h1>HCB is under maintenance</h1>
+        </header>
+        <p>Hang on! We'll be back soon :)</p>
+        <img class="dino" src="https://cloud-4q3ilvs9v-hack-club-bot.vercel.app/0dino_debugging.png" height="400px"/>
+        <br />
+        <span style="color: #8492a6;">Contact the HCB team at <a href="mailto:hcb@hackclub.com">hcb@hackclub.com</a></span>
+    </body>
+</html>
```
Comment (garyhtou) [inline docs/maintenance-mode.html]:
```suggestion
</html>

```

**Review by garyhtou — CHANGES_REQUESTED** [review]
Looks great! couple small changes

**Review by garyhtou — APPROVED** [review]
Great!

**Pull request merged**